lib/round.rb

```diff
@@ -0,0 +1,78 @@
+class Round
+  attr_reader :deck, :turns, :current_card, :current_turn, :number_correct,:cat_choice_qs, :correct_qs, :num_right
```

Comment:
There are a lot of `attr_reader`s here. We should be able to get away with just a few and use our methods for handling the rest of our data.